docs/modules/angular/pages/ngrx/cookbook/adding-search-criteria/general.adoc

```diff
@@ -100,6 +94,22 @@ _Adapt in File:_ `+<feature>-search.component.html+`
         ...
 ----
 
+* `*ocxAdvanced` directive
+
+The `*ocxAdvanced` directive specifies whether a field is part of the basic or advanced mode. This mode can be toggled using a button in the search header. The directive will be added to the div element
+
+[source, html]
+----
+ <form [formGroup]="criteriaGroup">
+      <div class="grid mt-0 p-fluid">
+        <div *ocxAdvanced class="col-12 md:col-3">
+          <span class="p-float-label">
+           // PLACE YOUR INPUT FIELD HERE
+          </span>
+        </div>
+        ...
+----
+
```

Comment:
This could be done in the separate angular/cookbook/directives/ocx-advanced/index.adoc, this would most likely also require angular/cookbook/components/search-header/index.adoc